### PR TITLE
fix(online users): Styled checkbox for online users

### PR DESCRIPTION
**Pull request by pavlovicnemanja92**
SD-2155

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

File: client/app/scripts/superdesk-desks/views/main.html [hunk 1]
```diff
@@ -3,12 +3,12 @@
     	<ul>
 	    	<li ng-repeat="v in views" ng-class="{active: v === view}">
 	    		<a href="" id="{{v}}" ng-click="setView(v)" translate>{{v}}</a>
-	    		<div ng-if="v === view && view === 'users'">
-	    			<input type="checkbox" ng-model="online_users" name="online_users" id="online_users" ng-change="changeOnlineUsers(online_users)">
-	    			<label translate>online users</label>
-	    		</div> 
+                        <div class="online-users" ng-if="v === view && view === 'users'">
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/views/main.html]: Can't we just say `v === 'users'`? :wink:

Comment (ioanpocol) [inline client/app/scripts/superdesk-desks/views/main.html]: No

Comment (plamut) [inline client/app/scripts/superdesk-desks/views/main.html]: Why not? The identity operator is transitive...

Comment (ioanpocol) [inline client/app/scripts/superdesk-desks/views/main.html]: Yes, you can do ng-if="v === view && v === 'users'" if you wish.
The point is that v and view should have value 'users' so just v==='users' is not enough.

Comment (plamut) [inline client/app/scripts/superdesk-desks/views/main.html]: Oh, OK, then it's fine of course, I thought it was just about the `v`. Thanks for clarifying!

**Pull request merged**